docs/calico-with-docker/AWS.md

```diff
@@ -173,13 +173,15 @@ aws ec2 run-instances \
   --key-name mykey \
   --security-group-ids $SECURITY_GROUP_ID \
   --user-data file://<PATH_TO_CLOUD_CONFIG>/user-data-first
-#  --subnet $SUBNET_ID 
+#  --subnet $SUBNET_ID
 #  Include the subnet param above if using a non-default VPC
+
+INSTANCE_ID_1=<InstanceId>
```

Comment:
TBH I'm not a fan of this syntax, I think having these additional steps just makes the doc longer (when the variables are generally used once), and people still have to copy the values around.  Every extra sentence in a doc reduces the chance that people will actually follow all the other ones by some tiny percentage, unmeasurable individually but very clear in aggregate.

If we want our docs to be shell scripts, then they should be shell scripts...